python/pyspark/pandas/groupby.py

```diff
@@ -2356,12 +2356,16 @@ def nunique(self, dropna: bool = True) -> FrameLike:
         Name: value1, dtype: int64
         """
         if dropna:
-            stat_function = lambda col: F.countDistinct(col)
+
+            def stat_function(col: Column) -> Column:
```

Comment:
let me just keep it as an explicit function def .. otherwise it complains about argument type `[Union[Column, str]]` that's not matched with `stat_function` below. I tried few other ways but seems like this is most straightforward.

Reply:
Oh... weird..